### DISCUSSION TITLE
你们是否计划支持OpenWrt

### DISCUSSION THREAD

**Discussion by AkarinLiu** [BeyondDimension/SteamTools#2694] [Q&A]
我翻看了你们的文档，发现有一个叫“openwrt.md”的文档，我不知道你们是否计划支持OpenWrt？
![image](https://user-images.githubusercontent.com/58081491/233236561-093dc694-be16-4dbe-9b23-8c2a2e1d4788.png)

**Comment by rmbadmin** [comment]
有计划支持

**Reply by yyyytawa** [reply]
您好,请问现在支持进度咋样了?
